source/globals.d.ts

```diff
@@ -1,6 +1,7 @@
 type AnyObject = Record<string, any>;
 
 declare const __featuresList__: string[];
+declare const __featureName__: 'webpack will replace this string';
```

Comment:
This is to avoid contributors trying to replace the `__featureName__` with the actual feature name.